File: Insertion-Sort/python/simple.py
import random;

arr = [random.randint(-20,50) for _ in range(10)]

print(arr)

def sort():
    for i in range(1,len(arr)):
        for j in range(i,0,-1):
            if arr[j-1] > arr[j]:
                arr[j-1],arr[j] = arr[j],arr[j-1]


sort()

print(arr)

File: Insertion-Sort/js/simple.js
const arr = []

function initData(count){
    for(let i = 0 ; i < count ; i++){
        arr.push(Math.floor(Math.random()*50-20))
    }
}

initData(10)

console.log(arr)

function sort(){
    let tmp
    for(let i = 1 ; i < arr.length ; i++){
        for(let j = i ; j > 0 ; j--){
            if(arr[j-1] > arr[j]){
                tmp = arr[j-1]
                arr[j-1] = arr[j]
                arr[j] = tmp   
            }
        }
    }
}

sort()

console.log(arr)